.github/workflows/release-step-1.yml

```
@@ -250,21 +251,23 @@ jobs:
             NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
         steps:
-            - name: Checkout public master branch
+            - name: Checkout master branch
               uses: actions/checkout@v3
               with:
                   fetch-depth: 0
+                  repository: ${{ github.repository }}
+                  token: ${{ secrets.MP_SEMANTIC_RELEASE_BOT }}
                   ref: master
 
             - name: Import GPG Key
-              uses: crazy-max/ghaction-import-gpg@v4
+              uses: crazy-max/ghaction-import-gpg@e00cb83a68c1158b29afc5217dd0582cada6d172 #v4.4.0
               with:
                   gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
                   passphrase: ${{ secrets.GPG_PASSPHRASE }}
                   git_user_signingkey: true
                   git_commit_gpgsign: true
 
-            - name: Merge release branch into master branch
+            - name: Merge pre-release branch into master branch
```

Sometimes when I read this, I get confused about the step because I think about the remote master branch.  I think adding local/remote would be valuable when referring to master/development:

```suggestion
            - name: Merge pre-release branch into local master branch
```